### PR TITLE
feat(base): creates project base and server

### PR DESCRIPTION
**Pull request by itsachen**
**What:**
This creates the SF movies project base and server.

**Why:**
This is part of the Miyagi engineering onboarding process.

**Description:**
- Creates a new NPM package with necessary dependencies
- Creates a Hapi server

### CODE REVIEW

**Comment by itsachen** [comment]
@charlesbjohnson

File: config/production.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+'use strict';
+
+module.exports = {
+  FOO: 'bar'
```
Comment (charlesbjohnson) [inline config/production.js]: Should this be `PORT`?

File: lib/index.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+'use strict';
```
Comment (charlesbjohnson) [inline lib/index.js:1]: nit: newline after this line

**Review by charlesbjohnson — APPROVED** [review]
LGTM pending some small stuff! 💯

File: config/production.js (hunk 1 above)
Comment (itsachen) [reply config/production.js]: I can change it for staging/production but at this point the tutorial explicitly mentions to change dev/test config files:

> We don't plan on deploying anything yet so the staging and production config variables do not need to be defined right now.

**Pull request closed without merging**